### ISSUE TITLE
Primary demand calculation fails for some converters

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etengine#484]
For the energy_distribution_cokes (ID: 645), primary and final demand are equal:

![](http://f.cl.ly/items/0N2o0m2C1N0s1u2v3200/Image%202012.10.30%203:27:14%20PM.png)

This is incorrect because the production of cokes (in the energy_cokesoven_transformation) has an efficiency of 92.8%. The primary energy calculation uses the 'loss' carrier to determine the primary demand. The losses of the cokes production process, however, are not calculated correctly (loss = 0.0).
